### PR TITLE
add HTTP/Delete method handler for SPIFFS

### PR DESCRIPTION
**Pull request by crossan007**
Support making HTTP/DELETE requests against `/static` and respond with JSON based on success/failure

### CODE REVIEW

**Comment by crossan007** [comment]
this will be used by https://github.com/crossan007/meshtastic-web/commit/1fe236ba7500782b7bb5861dd064617bf598eabb

File: src/meshwifi/meshhttp.cpp [hunk 1]
```diff
@@ -240,6 +241,7 @@ void initWebServer()
     ResourceNode *nodeRoot = new ResourceNode("/", "GET", &handleRoot);
     ResourceNode *nodeStaticBrowse = new ResourceNode("/static", "GET", &handleStaticBrowse);
     ResourceNode *nodeStaticPOST = new ResourceNode("/static", "POST", &handleStaticPost);
+    ResourceNode *nodeStaticDelete = new ResourceNode("/static", "DELETE", &handleStaticDelete);
```
Comment (mc-hamster) [inline src/meshwifi/meshhttp.cpp:244]: Request to move this to : /json/spiffs/delete/static

Comment (Noki) [reply src/meshwifi/meshhttp.cpp:244]: It probably would be wise to add password protection for all routes that change the SPIFFS and Settings, if we plan to make the web interface accessable in LAN environements.

Comment (crossan007) [reply src/meshwifi/meshhttp.cpp:244]: Good idea!  Can you open a new issue in which to track that feature?

Comment (mc-hamster) [reply src/meshwifi/meshhttp.cpp:244]: Building in access control is on Kevin’s todo list. This is going to be built into the main device API and then inherited by the http stack.

Comment (crossan007) [reply src/meshwifi/meshhttp.cpp]: @mc-hamster endpoint rename done :-)

**Pull request merged**